packages/core/getBusyTimes.ts

```diff
@@ -89,56 +89,54 @@ export async function getBusyTimes(params: {
       in: [BookingStatus.ACCEPTED],
     },
   };
-
-  // INFO: Refactored to allow this method to take in a list of current bookings for the user.
-  // Will keep support for retrieving a user's bookings if the caller does not already supply them.
-  // This function is called from multiple places but we aren't refactoring all of them at this moment
-  // to avoid potential side effects.
-  const bookings = params.currentBookings
-    ? params.currentBookings
-    : await prisma.booking.findMany({
-        where: {
-          OR: [
-            // User is primary host (individual events, or primary organizer)
-            {
-              ...sharedQuery,
-              userId,
-            },
-            // The current user has a different booking at this time he/she attends
-            {
-              ...sharedQuery,
-              attendees: {
-                some: {
-                  email: userEmail,
-                },
-              },
+  const matchedBookings = await prisma.booking.findMany({
```

Comment:
Storing it in `matchedBookings` now

Reply:
This is now always running the query which we don't want to do. This is what caused major slow downs for round-robin events and why we allowed the bookings to be passed into this function from the top-level `getSchedule` layer. This needs to be reverted.

Reply:
Amazing spot @keithwillcode 🙏 Completely missed it.